src/uk/gov/hmcts/contino/DockerImage.groovy

```diff
@@ -28,14 +28,16 @@ class DockerImage {
   Acr acr
   def commit
   def registryHost
+  def lastcommittime
 
 
-  DockerImage(product, component, acr, tag, commit) {
+  DockerImage(product, component, acr, tag, commit, lastcommittime) {
```

Comment:
camel case syntax please
```suggestion
  DockerImage(product, component, acr, tag, commit, lastCommitTime) {
```

normally we would make this compatible but doesn't look do-able unfortunately, let's see how it goes